src/components/CarrouselProfile/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+//Imports Material-ui
+import { Box, Typography } from "@material-ui/core";
+//Imports Components
+import CardCarrousel from "components/CardCarrousel";
+//Imports Styles
+import "./styles.scss";
+
+const CarrouselProfile = ({ titleSection, data }) => {
+  return (
+    <Box className="CarrouselProfile" component="article">
+      <Box component="header">
+        <Typography component="h2">{titleSection}</Typography>
+      </Box>
+      <Box>
+        {data.map(datos => (
```

Comment:
Estamos usando todos los nombres es inglés

```suggestion
        {items.map(item => (
```

Reply:
Done!